neural_compressor/torch/quantization/quantize.py

```diff
@@ -30,7 +30,7 @@ def need_apply(configs_mapping: Dict[Tuple[str, callable], BaseConfig], algo_nam
     return any(config.name == algo_name for config in configs_mapping.values())
 
 
-@log_quant_execution
+@log_quant_execution(mode=Mode.QUANTIZE.value)
```

Comment:
Hi @chensuyue, please be aware of this change, the quantization process is complete when either the `convert` or `quantize` is completed.